utils/envvars/parser.go

```diff
@@ -135,6 +142,33 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// GetSAASHost returns SAAS host env variable value.
+// if it's valid, otherwise returns defaultSAASHost.
+func GetSAASHost(env string) string {
+	if v, b := os.LookupEnv(envSAASHost); b {
+		if u := govalidator.IsURL(v); u {
+			logrus.Warnf("SAAS host changed to %s.", v)
+
+			return v
+		}
+		logrus.Warnf("environment variable %s has invalid format %s.", env, v)
```

Comment:
Here we will fallback to the `defaultSAASHost`. I'm not sure that it is a good idea, better to return error that will cause app exit. It will explicitly show that something configured wrong.

Reply:
Got it, should I throw Fatal here?

Reply:
I think better to return error `func GetSAASHost(env string) (string, error)`, and then service also will return error, then it will be logged in `main.go` with `Fatal`

Reply:
Done, thanks